operations/kubernetes/CronJobs.md

```diff
@@ -0,0 +1,32 @@
+# Kubernetes CronJobs
```

Comment:
Add an entry for this new file to the /operations/index.md file

Reply:
Will do, thanks for the review!

Reply:
added